pkg/executor/delete.go

```diff
@@ -83,10 +83,13 @@ func (e *DeleteExec) deleteSingleTableByChunk(ctx context.Context) error {
 		tbl           table.Table
 		isExtrahandle bool
 		handleCols    util.HandleCols
+		colPosInfo    *plannercore.TblColPosInfo
 		rowCount      int
 	)
-	for _, info := range e.tblColPosInfos {
+	for i := range e.tblColPosInfos {
```

Comment:
why make this change？

Reply:
I think `for i, info := range e.tblColPosInfos` is better

Reply:
removed the loop for this single table deletion case